post_info.php

```diff
@@ -0,0 +1,359 @@
+        // use database 'tafs'
+        $conn->exec("use tafs;");
+        // Execute the sql statement
+        $conn->exec($sql);
+
+        // Close the connection
+        $conn = null;
+
+        // return a 'success' status
+        return "success!";
+    } catch (PDOException $e) {
+        return $e->getMessage();
+        exit();
+    }
+}
+
+/**
+ * Return an id that is used for updating choices, survey_choice ...
+ * @param sql:string The select statement to be executed
+ * @return int: An id that is to be used
+ */
+function sql_get_data ($sql) {
+    require './db/config.php';
+
+    try {
+        // connect to the mysql database
+        $conn = new PDO(
+            "mysql:host=$servername;dbname=$database",
+            $username,
+            $password
+        );
+        // set the PDO error mode to exception
+        $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+        // use database 'tafs'
+        $conn->exec("use tafs;");
+        // Create an array to store the returned data
+        $return_array = array();
+        // Execute the sql statement and get the id back
+        foreach($conn->query($sql) as $row) {
+            for ($i = 0; $i < count($row); $i++) {
+                array_push($return_array, $row[$i]);
+            }
+        }
+        $conn = null;
+        return $return_array;
+    } catch (PDOException $e) {
+        return $e->getMessage();
+        exit();
+    }
+}
+
+/* ---------- Encapsulate logic for the operations -------------- */
+
+/**
+ * helper function for executing the SQL statement of updating user_info
+ * @param user_list:array An array contain a list of user information with
+ *                  user_id, name,
+ */
+function handle_user_info ($user_list) {
+    // Initialize a success status array
+    $return_data = Array();
+    // print_r($data);
+    //print_r($data["user_list"]);
+    //var_dump($data_process[0]["user_list"]);
+    for ($i = 0; $i < count($user_list); $i++) {
+        $user_id = $user_list[$i]->user_id;
+        $name = $user_list[$i]->name;
+        $photo = $user_list[$i]->photo;
+        $sql = query_update_user_info($action, $user_id, $name, $photo);
+        // Operate the database according to the sql generated
+        $status = execute_sql($sql);
+        // Determine the status and then add the status object in the returned array
+        if ($status == "success!") {
+            $temp = array('TYPE' => 'success', 'data' => null);
+            //$temp->TYPE = 'success';
+            //$temp->data = null;
+            array_push($return_data, $temp);
+        } else {
+            $temp = new \stdClass();
+            $temp->TYPE = "error";
+            $temp->data = $status;
+            array_push($return_data, $temp);
+        }
+    }
+    // Encapsulate the data into a php object
+    $return_json = new \stdClass();
+    $return_json->TYPE = "success";
+    $return_json->data = $return_data;
+    echo json_encode($return_json);
```

Comment:
You must `exit` after JSON is echoed otherwise the rest of the script might echo somthing else producing invalid JSON